src/index.html

```diff
@@ -17,16 +17,22 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form data-qa="big">
+      <input
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+        class="input big-input"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form data-qa="small">
+      <input
+        type="text"
+        data-qa="hover"
```

Comment:
The `data-qa` attribute is set to `hover`, but it should be `search-bar-small` according to the task requirements.